geode-core/src/main/java/org/apache/geode/management/cli/CliFunction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.management.cli;
+
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.internal.cache.execute.InternalFunction;
+import org.apache.geode.management.internal.cli.functions.CliFunctionResult;
+
+/**
+ * An adapter interface to be implemented by cli functions and to return CliFunctionResult.
```

Comment:
Let's keep the terminology consistent. This should not be referred to as an interface. 

Also amend the commit message to not call this an interface.

Reply:
Thanks @pdxrunner. I have changed it to call it as a class and will remember to update the commit message while squashing and merging to develop